lib/screens/account.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:ofypets_mobile_app/scoped-models/main.dart';
+import 'package:ofypets_mobile_app/screens/auth.dart';
+import 'package:ofypets_mobile_app/screens/order_history.dart';
+import 'package:ofypets_mobile_app/screens/change_email.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:ofypets_mobile_app/utils/constants.dart';
+import 'package:http/http.dart' as http;
+
+class Account extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Account"),
+      ),
+      body: Container(
+        child: account_options(),
+      ),
+    );
+  }
+
+  Widget account_options() {
+    return ScopedModelDescendant(
+        builder: (BuildContext context, Widget child, MainModel model) {
+      return ListView(
+        children: <Widget>[
+          ListTile(
+            title: Text(
+              "Order History",
+              style: TextStyle(fontWeight: FontWeight.w600),
+            ),
+            onTap: () {
+              navigate_option("order_history", context, model);
+            },
+          ),
+          ListTile(
```

Comment:
Do Comment this, so that user doesn't get frustrated by tapping on it as it will not respond.